dev/com.ibm.ws.app.manager.module/src/com/ibm/ws/app/manager/module/internal/DeployedAppInfoBase.java

```diff
@@ -118,15 +122,137 @@ public List<ContainerInfo> getClassesContainerInfo() {
         }
     }
 
+    // Data for caching a container ...
+
+    protected static class LibraryContainerData {
```

Comment:
Protected in case a subclass has a need to use this data structure.
'public final' with no accessors to keep the class lightweight.  It's intended to be a lightweight record type data structure.